src/tritonserver.cc

```diff
@@ -646,7 +646,7 @@ TRITONSERVER_LogIsEnabled(TRITONSERVER_LogLevel level)
 TRITONAPI_DECLSPEC TRITONSERVER_Error*
 TRITONSERVER_LogMessage(
```

Comment:
You're going to want to be really careful here. If you're changing API functions, you have to update the minor version. But this seems like a breaking change (i.e. would need to be a major version update, which you want to avoid for a small change), given that it changes how the API is called even with a default value. For example, in stubs. It may make more sense to add a separate function.